go/inst/instance_topology.go

```diff
@@ -2860,7 +2860,11 @@ func relocateReplicasInternal(replicas [](*Instance), instance, other *Instance)
 	}
 
 	// Too complex
-	return nil, log.Errorf("Relocating %+v replicas of %+v below %+v turns to be too complex; please do it manually. Additional Errors: %v", len(replicas), instance.Key, other.Key,gtidErrorsMsg), errs
+	// if the len of gtidErrorsMsg is less than 21, no errors were added
+	if len(gtidErrorsMsg) < 21 {
```

Comment:
Why 21?
I suppose it is the check if something was added after 'Additional Errors: ' string.
If yes, my proposal is to change the implementation to
1. Intialize gtidErrorsMsg to empty string at the beginning (line 2817)
2. Do the job
3. check if len(gtidErrorMsg) > 0. If yes, do gtidErrorsMsg = "Additional Errors: " + gtidErrorsMsg

Reply:
Modified according to the above comments, passed Jenkins tests.